Makefile

```
@@ -211,6 +211,12 @@ check.docs:
 	@$(MAKE) docs
 	@tests/scripts/validate_modified_files.sh docs
 
+check.codegen:
+	@$(MAKE) codegen
+	@tests/scripts/validate_modified_files.sh codegen
```

Any reason this reruns make instead of setting codegen as a dependency of `check.codegen`?

```suggestion
check.codegen: codegen
	@tests/scripts/validate_modified_files.sh codegen
```

---

@BlaineEXE  wrote:

> Any reason this reruns make instead of setting codegen as a dependency of `check.codegen`?

Good point. I considered that but  was following the pattern of `check.docs` and `check.helm-docs` instead. But I will apply  your suggestion.